app/presenters/completions_presenter.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
 class CompletionsPresenter
+  include Rails.application.routes.url_helpers
   include ActionView::Helpers::TranslationHelper
   include ActionView::Helpers::TagHelper
 
   attr_reader :current_user, :current_sp, :decrypted_pii, :requested_attributes,
-              :completion_context, :selected_email_id
+              :completion_context, :selected_email_id, :url_options
```

Comment:
As implemented `url_options` will always be `nil`. `url_options` includes the locale, so the effect is that the paths generated below will always be for the English URL, even if the user is viewing the site in another language.

We should update the constructor to receive `url_options:` as an argument and pass that from the controller where it's initialized.

Reply:
This isn't quite right still. We need to pass `url_options` when constructing `CompletionsPresenter` from the controller:

```diff
diff --git a/app/controllers/sign_up/completions_controller.rb b/app/controllers/sign_up/completions_controller.rb
index 76fb9b252d..14977baf25 100644
--- a/app/controllers/sign_up/completions_controller.rb
+++ b/app/controllers/sign_up/completions_controller.rb
@@ -56,2 +56,3 @@ module SignUp
         selected_email_id: user_session[:selected_email_id_for_linked_identity],
+        url_options:,
       )
```